### PR TITLE
[Docs] How to write a new runner

### LINKED ISSUE #723: Write manual for writing a new runner
https://github.com/sider/runners/blob/20228d6da51c80378d15b8c30cf876caa7ede8c5/docs/how-to-write-a-new-runner.md#L3

### PR DESCRIPTION
**Pull request by ybiquitous**
Fix #723

### CODE REVIEW

**Comment by ybiquitous** [comment]
@yykamei Could you please review this lightly if you have time?

File: docs/how-to-write-a-new-runner.md [hunk 1]
```diff
@@ -1,3 +1,152 @@
...
+
+### Create a base image
+
+If it is not provided yet, create a Docker base image on [sider/devon_rex](https://github.com/sider/devon_rex) first.
+
+A runner image is built based on a base image. We manage base images in [sider/devon_rex](https://github.com/sider/devon_rex) repository.
+These images are prepared for each programming environment (e.g. Ruby, Java, PHP, and so on).
+
+We have already provided many base images. Thus, you do not need to add a new base image in almost cases.
+However, you need to make a new one when you try to add a new programming language and environment.
```
Comment (yykamei) [inline docs/how-to-write-a-new-runner.md]: This description is good, but I think we should explain about `devon_rex` first.
How about this?

> The images of Runners inherit from our Docker base images on [sider/devon_rex](https://github.com/sider/devon_rex), and each of them provides a specific programming environment (e.g., Ruby, Java, PHP, and so on).
> 
> You don't have to create new base images in almost cases. However, if your new runner requires a new programming environment, you would have to get into building the new base image first.

Comment (ybiquitous) [reply docs/how-to-write-a-new-runner.md]: Sounds good. 👍 
I will change it.

Comment (ybiquitous) [reply docs/how-to-write-a-new-runner.md]: Done. 5c66080fd928ad12410e939f1582e58160003a9d

**Pull request merged**